### DISCUSSION TITLE
Command "caddy" not found

### DISCUSSION THREAD

**Discussion by enpro-github** [Budibase/budibase#4534] [Help] [answered]
I'm trying to setup a reverse proxy with caddy, I have installed it no problem (https://docs.budibase.com/self-hosting/hosting-settings/advanced-options/reverse-proxy) but after I do I am unable to run `caddy` or any other caddy command because it said it cannot be found. I don't believe there's any issue with my environment variables

**Comment by shogunpurple** [comment, marked as answer]
Can we assume that this is now resolved given you are running caddy commands here?
https://github.com/Budibase/budibase/discussions/4536

**Reply by enpro-github** [reply]
I have fixed the issue by using the command ` docker run cmd.cat/caddy caddy ` instead, not sure how to get it working with just `caddy`.

**Reply by JCV67** [reply]
I am experiencing the same problem on Docker under Ubuntu. After 'docker pull caddy' the result of de command 'caddy' is command not found.

**Reply by albotroz** [reply]
I have still the same problem. Is doc up-to-date or even correct? https://caddyserver.com/docs/command-line#caddy-run
